frontend/src/components/SettingsDialog.tsx

```diff
@@ -49,7 +67,7 @@ function SettingsDialog({ settings, setSettings }: Props) {
         <div className="flex items-center space-x-2">
           <Label htmlFor="image-generation">
             <div>DALL-E Placeholder Image Generation</div>
-            <div className="font-light mt-2 text-xs">
+            <div className="mt-2 text-xs font-light">
```

Comment:
Are these linter changes? What are you using? 

And please revert it as part of this PR. We can make style changes but it shouldn't be mixed into a PR that adds functionality.

Reply:
Needs to be obvious whether Tailwind config is enabled, and need to provide a way to remove the previously added Tailwind config or toggle it off.